### DISCUSSION TITLE
Control surface button issue

### DISCUSSION THREAD

**Discussion by KingMichaelCoy** [tttapa/Control-Surface#803] [General]
Hi 

I am using the send midi notes code for my midi controller which is assigned to pin 22 of my Adafruit huzzah32. How do I get more than this one button to work. I have other pins assigned to the other five buttons but when I try to assign to the other corresponding buttons it does not trigger the other buttons. How do I make multiple buttons trigger midi notes and not just what's assigned to pin 22? Do I need a multiplexer? I do have one but it seems it should work without using this.

**Comment by Veroledez** [comment]
Hola, pública tu codigo, asi es mas facil comprender tu problema

**Comment by tttapa** [comment]
As mentioned by Veroledez, please post your code.

> Do I need a multiplexer?

No, a multiplexer is not required.
